src/components/common/useRequest.ts

```diff
@@ -53,3 +55,7 @@ export function useRequest<T = unknown>(
     refreshWhenOffline: true,
   });
 }
+
+function isCloudWorkstation(url: string) {
+  return url.includes('cloudworkstations.dev');
```

Comment:
I think this should be `.endsWith`
```suggestion
  return url.endsWith('.cloudworkstations.dev');
```

Reply:
Nope, url can include paths - ie: '9099-blah.cluster.cloudworkstations.dev/auth'

Reply:
nit: I think my preference is that we call this with the hostname path if we have it and therefore have `endsWith` just so that it's a stronger validation